packages/playwright-core/src/server/webkit/wkBrowser.ts

```diff
@@ -267,7 +271,10 @@ export class WKBrowserContext extends BrowserContext {
     const cc = network.rewriteCookies(cookies).map(c => ({
       ...c,
       session: c.expires === -1 || c.expires === undefined,
-      expires: c.expires && c.expires !== -1 ? c.expires * 1000 : c.expires
+      expires: c.expires && c.expires !== -1 ? c.expires * 1000 : c.expires,
+      // With "Prevent cross-site tracking." on by default in Safary, SameSite=None cookies
+      // behave as Lax.
+      sameSite: (!c.sameSite || c.sameSite === 'None') ? 'Lax' : c.sameSite,
```

Comment:
same here, cookie jar should contain raw value, should write None if it was specified here just fine.
Actual behavior of SameSite=None will be determined at runtime, depending on the tracking flag.